src/screens/UserPortal/Pledges/Pledges.tsx

```
@@ -238,8 +234,8 @@
               <div
                 className={styles.moreContainer}
                 aria-describedby={id}
-                data-testid="moreContainer"
-                onClick={(e) => handleClick(e, users.slice(2))}
+                data-testid={`moreContainer-${params.row.id}`}
+                onClick={() => handleClick(users.slice(2))}
               >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)